src/jobservice/main.go

```diff
@@ -57,6 +60,9 @@ func main() {
 		panic(err)
 	}
 
+	cfgLib.InitTraceConfig(ctx)
+	defer tracelib.InitGlobalTracer(context.Background()).Shutdown()
```

Comment:
defer should move to the entry of this func.

Reply:
place it here because of reuse the ctx